File: api/nodejs/app/models/student.js
var mongoose     = require('mongoose');
var Schema       = mongoose.Schema;

var StudentSchema = new Schema({
	FULLNAME: String,
    ADDRESS: String,
    TELEPHONE: String,
    BIRTHPLACE: String,
    BIRTHDATE: Date,
    GENDER: String,
    RELIGION: String,
    PARENTNAME: String,
    PARENTADDRESS: String,
    PARENTPHONE: String,
    PARENTJOB: String,
    PICTURE: String,
    CREATEDAT: Date,
    UPDATEAT: Date
});

module.exports = mongoose.model('Student', StudentSchema);

File: api/go/models/entity/student.go
package entity

import "time"

type Gender int

const (
	MALE_GENDER Gender = iota
	FEMALE_GENDER
)

type Religion int

const (
	ISLAM_RELIGION Religion = iota
	KRISTEN_RELIGION
	CATHOLIC_RELIGION
	HINDU_RELIGION
	BUDHA_RELIGION
)

type Student struct {
	BaseModel
	Fullname      string
	Address       string
	Phone         string
	Birthplace    string
	Birthdate     time.Time
	Gender        Gender
	Parentname    string
	Parentaddress string
	Parentphone   string
	Parentjob     string
	Picture       string
	Religion      Religion
}
